libraries/AP_Math/AP_Math.cpp

```diff
@@ -32,7 +32,12 @@ is_equal(const Arithmetic1 v_1, const Arithmetic2 v_2)
         return fabs(v_1 - v_2) < std::numeric_limits<double>::epsilon();
     }
 #endif
+#pragma GCC diagnostic push
```

Comment:
Should this be `#pragma clang` rather then GCC?

Reply:
Fixed.